_modules/ceph_cfg/__init__.py

```diff
@@ -217,6 +217,33 @@ def osd_reweight(**kwargs):
     return ceph_cfg.osd_reweight(**kwargs)
 
 
+def osd_df(**kwargs):
+    """
+    OSD disk space report
+
```

Comment:
Do osd_df and cluster_df differ in what they report? Other then one OSD vs. all OSDs? If not, why not make the docs consistent? Maybe 'Get an OSDs free space'

Or does osd_df report other things too? Then a different name might be appropriate?